exercises/ruby-1-a/ruby_1_a_example_1.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class ShoppingCart
+  STOCK = {
+    STAPOT: ['Potatoes', 10.0],
+    STARIC: ['Rice', 30.00],
+    STACOF: ['Coffee', 14.99],
+    MEDNEW: ['Newspaper', 2.99]
```

Comment:
The SKU numbers used here should be reflected in the instructions.  No value in not giving this as factual data.

Reply:
Yep, now the README also reflects these changes